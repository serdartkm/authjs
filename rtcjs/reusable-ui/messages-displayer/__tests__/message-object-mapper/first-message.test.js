import{h} from 'preact'
import  render from 'preact-render-to-string';
import FirstMessage from '../../lib/message-object-mapper/first-message'

describe("SNAPSHOT TEST FirstMessage component snapshot",()=>{
    it("Message local renders correctly",()=>{
        const tree =render(<FirstMessage FirstMessage datetime={1569393960355} message="First Message" letter="m" local />)
         expect(tree).toMatchSnapshot();
    })

    it("Message remote renders correctly",()=>{
        const tree =render(<FirstMessage FirstMessage datetime={1569393960355} message="First Message" letter="m" local={false} />)
     expect(tree).toMatchSnapshot();
    })
})