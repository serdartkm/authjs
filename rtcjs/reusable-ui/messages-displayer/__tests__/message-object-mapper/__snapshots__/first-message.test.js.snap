// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SNAPSHOT TEST FirstMessage component snapshot Message local renders correctly 1`] = `"<div style=\\"display: flex; align-items: center;\\"><div style=\\"background-color: #FFECB3; padding: 5px; margin: 2px; border-radius: 15px; border-color: #9E9E9E; border-style: solid; border-width: 2px; max-width: 100%; word-wrap: break-word; word-break: break-all; min-width: 30%;\\"><div>First Message</div><div style=\\"font-size: 10px; padding-top: 2px; text-align: end;\\"><i style=\\"background-color: #efebe9;\\">11:46:00 AM</i></div></div></div>"`;

exports[`SNAPSHOT TEST FirstMessage component snapshot Message remote renders correctly 1`] = `"<div style=\\"display: flex; align-items: center;\\"><div style=\\"height: 30px; width: 40px; padding: 3px; border-radius: 30px; background-color: darkSmoke; border-style: solid; border-width: 2px; display: flex; justify-content: center; align-items: center; color: #009688; border-color: #80cbc4;\\"><div>M</div></div><div style=\\"background-color: #FFECB3; padding: 5px; margin: 2px; border-radius: 15px; border-color: #9E9E9E; border-style: solid; border-width: 2px; max-width: 100%; word-wrap: break-word; word-break: break-all; min-width: 30%;\\"><div>First Message</div><div style=\\"font-size: 10px; padding-top: 2px; text-align: end;\\"><i style=\\"background-color: #efebe9;\\">11:46:00 AM</i></div></div></div>"`;
